### DISCUSSION TITLE
Release date of v1.0.0

### DISCUSSION THREAD

**Discussion by nqb** [ovh/venom#346] [General]
Hello,

I would like to know if you already schedule release date of v1.0.0 (stable).

Thanks for your feedback.

**Comment by yesnault** [comment]
Hi,

Probably at the end of January if there is no bug detected

**Reply by nqb** [reply]
Thanks for your feedback @yesnault.

**Comment by FAC-1312** [comment]
Hello, I would like to know what I need to know if I'm taking in hands a suite of tests created on march 2020. There is some essential updating to do or there's no reason for actual test fails?

**Reply by yesnault** [reply]
you have to read the changelog before upgrade venom to 1.0.0-x, see https://github.com/ovh/venom/releases/tag/v1.0.0-beta.2

**Reply by FAC-1312** [reply]
Okay but it's mandatory? I mean I must to upgrade my version or can I continue with current installed version ? I'm taking the work of a colleague and some old tests now are fail. Could this fails be related with the version in my project?

**Reply by kevinramage** [reply]
Hello,
I check with Fernando the issue, there are no link between Fernando issue and new venom version. @Fernandoj2ee, you can continue to work with the venom fork version.
Regards

**Comment by yesnault** [comment]
FYI https://github.com/ovh/venom/releases/tag/v1.0.0-beta.4 available with some fixes -- cc @nqb
